imports/plugins/included/default-theme/client/styles/select.less

```diff
@@ -25,3 +25,7 @@
   width: 44px;
   pointer-events: none;
 }
+
+.Select-menu-outer {
```

Comment:
Yeah it shouldn't but this class-name is generated by `react-select` for all the dropdowns. Hence the capital letter